src/shared/components/studyTagsTooltip/StudyTagsTooltip.tsx

```diff
@@ -64,7 +64,7 @@ export default class StudyTagsTooltip extends React.Component<StudyTagsTooltipPr
         if (meta.isComplete) {
             const resultKeyLength = Object.keys(meta.result).length;
             const description = <div dangerouslySetInnerHTML={this.addHTMLDescription(this.props.studyDescription)}/>;
-            overlay = resultKeyLength > 0 ? ([description, <br/>, <div className="studyTagsTooltip"> <JsonToTable json={meta.result}/></div>]) : description;
+            overlay = description; // resultKeyLength > 0 ? ([description, <br/>, <div className="studyTagsTooltip"> <JsonToTable json={meta.result}/></div>]) : description;
```

Comment:
@oplantalech i disabled tags in order to prevent import of jsonToTable library until library dependency issues can be resolved.